task9/config/config.ts

```diff
@@ -0,0 +1,23 @@
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+interface Config {
+  port: number
+  startDate: string
+  endDate: string
+  nasaMeteorUrl: string
+  nasaRoverUrl: string
+  sol: number
+}
+
+const config: Config = {
+  port: parseInt(process.env.APP_PORT || '4000'),
```

Comment:
`parseInt(process.env.APP_PORT) || 4000` looks better

Reply:
do you need this file?

Reply:
no, we don't and I'm not using it, that's why I asked you check specific commit )